test/data/validator.input.yaml

```diff
@@ -1,6 +1,9 @@
 yaml: |
   jobs:
       main:
+          annotations:
+              beta.screwdriver.cd/auto_pr_builds: fork-only
+          description: "This is a description"
```

Comment:
I'm kinda confused. Shouldn't this be in the same level as settings and environments?